charts/scalardb-analytics-postgresql/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: scalardb-analytics-postgresql
+description: ScalarDB Analytics with PostgreSQL
+type: application
+version: 1.0.0-SNAPSHOT
+appVersion: 3.10.3
```

Comment:
At this time, ScalarDB Analytics with PostgreSQL does not release the `SNAPSHOT` version. So, I set the latest stable version.

However, we are working on releasing the `SNAPSHOT` version on the ScalarDB Analytics with PostgreSQL side. In the future, we will set the `SNAPSHOT` version here.